driver/examples/configs/test_external_C12_1x1.yaml

```diff
@@ -0,0 +1,102 @@
+stencil_config:
+  compilation_config:
+    backend: numpy
+    rebuild: false
+    validate_args: true
+    format_source: false
+    device_sync: false
+grid_config:
+  type: external
+  config:
+    grid_type: 0
+    grid_file_path: "../../test_input/C12.tile"
```

Comment:
This not part of the PR. This wasn't picked up by the tests because the 54 ranks tests are not running right now!

Reply:
Is there way I can include them? Should we be running the 54 rank tests?

Reply:
How big are the files? Technically I think it should be located at `driver/examples/data` so that the example folder is indeed self standing.

We should be running 54 ranks but I haven't look at if we can do that on the `Github` free runners without it taking 8 million years. Someone has to go ahead and make a github action and test it out

Reply:
We can start a plan to set up a CI which uses cloud resources from the NOAA cloud resources provider.  We've done this successfully for SHiELD, FMS, and other GFDL repos.  The key is to make sure we have a container with the needed bits.